pyproject.toml

```diff
@@ -65,4 +65,7 @@ preview = true
 pyink-indentation = 2
 pyink-use-majority-quotes = true
 
-[tool.setuptools_scm]
\ No newline at end of file
+[tools.setuptools.dynamic]
+version = {attr = "xee.__version__"}
+
+[tool.setuptools_scm]
```

Comment:
I'm not sure we should take this approach. Can we try generating the version file with setuptools_scm instead? 

https://github.com/pypa/setuptools-scm

```
[tool.setuptools_scm]
version_file = "pkg/_version.py"
```

Will that fix the issue with conda that lead to this patch?

Reply:
I never use setuptools_scm, I personnaly use commitizen to generate the version numbers. I think from conda the version number needs to exist at least in 1 place that does not require to load the modules. If this does the job as well I'm happy with it, I just don't know how to test it if not doing patch releases.

Reply:
I wonder how other folks solve this problem. I'd like to keep using setuptools_scm if possible and not have two places where that is traced (version control and code). It looks like Xarray does something similar. Maybe, to address the conda issue, we could do what they do and set a fallback version?

https://github.com/pydata/xarray/blob/1c6300c415efebac15f5ee668a3ef6419dbeab63/pyproject.toml#L70

Reply:
Let me do that in another PR and see if it works